test-em-all.bash

```
@@ -0,0 +1,242 @@
+then
+  echo "Restarting the test environment..."
+  echo "$ docker-compose down --remove-orphans"
+  docker-compose down --remove-orphans
+  echo "$ docker-compose up -d"
+  docker-compose up -d
+fi
+
+waitForService curl http://$HOST:$PORT/actuator/health
+
+# Verify access to Eureka and that all four microservices are registered in Eureka
+assertCurl 200 "curl -H "accept:application/json" $HOST:$PORT/eureka/api/apps -s"
```

Shellcheck (0.10.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)